src/todolist/Task.py

```diff
@@ -27,8 +27,8 @@ def from_trello(cls, item, status=Status.ToDo):
         task.last_modified = datetime.strptime(item['dateLastActivity'], dateformat)
         return task
 
-    def touchedToday(self):
-        return self.last_modified.date() == datetime.now().date()
+    def touchedOn(self, date):
```

Comment:
This should be reverted. If you are adding function parameters for testing reasons, something's probably wrong.

Reply:
I will revert most of this and have another go.
It's probably worth me feeding back that I was trying to apply the same restructuring as in the example on pgs 25-27 of the reading material for module 3. I've clearly misapplied it, but worth mentioning that's where I got it from.